[Auditbeat] [6.8] Linux build flaky integration test

Looking at the build history in https://beats-ci.elastic.co/job/elastic+beats+6.8+multijob-linux/beat=auditbeat,label=linux-immutable/, some builds have failed with errors like this:

```
13:37:19 Cannot create container for service beat: invalid CapAdd: unknown capability: "CAP_AUDIT_READ"
13:37:19 make[1]: *** [integration-tests-environment] Error 1
13:37:19 make[1]: Leaving directory `/var/lib/jenkins/workspace/elastic+beats+6.8+multijob-linux/beat/auditbeat/label/linux-immutable/src/github.com/elastic/beats/auditbeat'
13:37:19 make: *** [testsuite] Error 2
```

However, other 6.8 Linux Auditbeat builds have succeeded. So there appears to be some flakiness here.

Also note that this is only happening with the 6.8 build. It is not observed on `master`.

--- a/auditbeat/docker-compose.yml
+++ b/auditbeat/docker-compose.yml
@@ -19,7 +19,6 @@ services:
     pid: host
     cap_add:
       - AUDIT_CONTROL
-      - AUDIT_READ
 
   # This is a proxy used to block beats until all services are healthy.
   # See: https://github.com/docker/compose/issues/4369

--- a/x-pack/auditbeat/docker-compose.yml
+++ b/x-pack/auditbeat/docker-compose.yml
@@ -10,4 +10,3 @@ services:
     pid: host
     cap_add:
       - AUDIT_CONTROL
-      - AUDIT_READ

